src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java

```diff
@@ -50,9 +50,17 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
                 return true;
             }
         }
-
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+        
+        
+        if (!player.isOp())
++        {

```

Comment:
Random plus signs? Do you even use a IDE to compile your code. What about testing?

Will you _please_ read the contributing guidelines!